package.json

```diff
@@ -27,6 +25,7 @@
   "author": "Mozilla (https://mozilla.org/)",
   "license": "MPL-2.0",
   "dependencies": {
+    "@dannycoates/nps": "^5.9.12",
```

Comment:
Were you going to try and submit a PR for your changes to nps?

Reply:
https://github.com/sezna/nps/pull/220

Reply:
This has been merged now.